spinalcordtoolbox/labels.py

```diff
@@ -278,24 +305,13 @@ def compute_mean_squared_error(img: Image, ref: Image) -> float:
     """
     coordinates_input = img.getNonZeroCoordinates()
     coordinates_ref = ref.getNonZeroCoordinates()
-
-    # check if all the labels in both the images match
-    if len(coordinates_input) != len(coordinates_ref):
-        raise ValueError(f"Input and reference image don't have the same number of labels! {len(coordinates_input)} vs {len(coordinates_ref)}")
-
-    rounded_coord_ref_values = [np.round(c.value) for c in coordinates_ref]
-    rounded_coord_in_values = [np.round(c.value) for c in coordinates_input]
     result = 0.0
 
+    # This line will add warning in the log if there are missing label.
+    find_missing_label(img, ref)
```

Comment:
It might be good to add this to indicate the function returns something, but we're just not using it. 

```suggestion
    _, _ = find_missing_label(img, ref)
```